bin/run_on_all_chains.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# Colors for the terminal output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+# Disable immediate exit on non-zero status
+set +e
```

Comment:
It's disabled by default, but I'd recommend to enable it.
Also it's better to do in this way:
```
  set -o errexit
  set -o pipefail

```

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3025/commits/de4b00303be7ac60113920161c73c183729e2412

Reply:
Reverting in https://github.com/matter-labs/zksync-era/pull/3025/commits/bdaf5e9ab1c397503b398802fca6cb3021805b44

Within the script we want to wait until all background processes complete their execution. Adding these flags will make the whole script fail early.